crates/tap-agent/src/agent/sender_accounts_manager.rs

```diff
@@ -72,25 +83,42 @@ impl Display for AllocationId {
     }
 }
 
+/// Enum containing all types of messages that a [SenderAccountsManager] can receive
 #[derive(Debug)]
 pub enum SenderAccountsManagerMessage {
+    /// Spawn and Stop [SenderAccount]s that were added or removed
+    /// in comparision with it current state and updates the state
```

Comment:
```suggestion
    /// in comparison with it current state and updates the state
```

This line isn't clear. Can you say what you mean another way? We'll figure this out! 💪